demos/security_barrier_camera_demo/README.md

```diff
@@ -96,6 +96,12 @@ To do inference for two video inputs using two asynchronous infer request on FPG
 ./security_barrier_camera_demo -i <path_to_video>/inputVideo_0.mp4 <path_to_video>/inputVideo_1.mp4 -m <path_to_model>/vehicle-license-plate-detection-barrier-0106.xml -m_va <path_to_model>/vehicle-attributes-recognition-barrier-0039.xml -m_lpr <path_to_model>/license-plate-recognition-barrier-0001.xml -d HETERO:FPGA,CPU -d_va HETERO:FPGA,CPU -d_lpr HETERO:FPGA,CPU -nireq 2
 ```
 
+To do inference for video inputs on Intel® Vision Accelerator Design with Intel® Movidius™ VPUs, somes optimization hints are suggested to make good use of the computation ability. Configuring the number of allocated frames (-n_iqs) to provide enough inputs for inference. Configuring the number of infer request (-nireq) to achieve asynchronous infer. Configuring the number of threads (-n_wt) for multi thread processing. For example, to run the sample on one Intel® Vision Accelerator Design with Intel® Movidius™ VPUs Compact R card, run the following command:
```

Comment:
somes -> some

Also, for better readability, I suggest to turn the middle of this paragraph into a list, e.g.:

* Configuring the number of allocated frames [...]
* Configuring [...]
* Configuring [...]